_posts/informational-interviews/2020-02-05-equity-4-who.md

```diff
@@ -0,0 +1,110 @@
+            - important to serve racialized students.
+            - original mission: counter institutional history of UW re anti-black and anti-indigenous behavior.
+        - education. teach-ins. online rn.
+    - all money from gofundme going into personal bank acct and e-transfered to community members
+    - tax issues for individual? looked into incorporating, charity, nonprofit, but high burden
+        - 5 students + 1 alumni. no expertise in finance/law and all the steps
+    - meaghan: heard about "fiscal sponsorship". contacted UK org. referred to OpenCollective. pointed to us.
+        - pc: what's platform in uk? #ask
+
+- Hypha: Patcon, Dawn, Ben
+- dc: similar trajectory, re: finances. took us a lot of time to figure out how to hold money.
+- dc: we are based out of Toronto/GTA. fulltime student at UofT. seen similar things. agree with failure to support.
+- pc: we're in tech, but technoskeptical
+- bl: background in eng, trying to do finance and accounting. not a lot of canadian context for new ways of doing fiscal sponsorshop (e.g. opencollective). figuring it out. first in canada. have had pro bono legal review, so reasonably confident in our approach. here for logistal questions.
+
+- Meaghan: read the guide, do have questions
+    - fees and expenses? 5% for Hypha
+    - not moving a lot of money. a couple thousand. quick turnaround.
+    - doing fiscal sponsorship bc don't want to spend lots on lawyers
+        - eager to minimize fees, but ack that there are costs. want to get as much as possible to recipients.
+- ben: fiscal sponsorship overview + breakdown
+    - fiscal sponsorship can mean a few things.
+    - can be very heavy. taking on legal contracts like leases. NOT what we're doing
+    - Hypha dealing in **donations, NOT services**. we're starting with support for no-strings donations, so no taxes need be charged. but using our system for transactions like selling event tickets would be different (taxes).
+    - need records for payouts. claims can be through OC website. needs to submit receipts/invoices. accountant needs them.
+    - meaghan: current = excel spreadsheet.
+        - requested funds. uw email. $38,000 in backlog of requests. working through.
+    - payout ways we currently use: bank transfer, and paypal.
+    - pc: can we do e-transfer? bl: requires more clicking, and our bank is not great.
+        - #update confirmed later than only one member of our co-op (yj) can perform e-transfers, so that's too complicated for us.
+    - bl: fees. **money in** = 5%. **money out** = no extra charge.
+        - #update bl realized there were still processing charges on payout (which e-transfer may not have), but just minimum of payment platform, e.g. paypal (nothing taken by OC platform or Hypha host)
+        - 5% breakdown
+            - 15% of that 5% goes to our platform (OC)
+            - ~85% of that 5% goes to our accountant
+            - ~10% to us (very low)
```

Comment:
@benhylau did i capture your realization properly (about money out costs)?

@benhylau also, my breakdown math doesn't add up. can you confirm how you explained?

Reply:
85% of that 5% goes to us mainly to pay our accountant

We do not know if we will operate at a loss or gain, our accountant bills by hour not a % of how much money gets processed.

Reply:
```suggestion
    - bl: fees. **money in** = 5%. **money out** = no extra charge.
        - #update bl realized there were still processing charges on payout (which e-transfer may not have), but just minimum of payment platform, e.g. paypal (nothing taken by OC platform or Hypha host)
        - 5% breakdown
            - 15% of that 5% goes to our platform (OC)
            - ~85% of that 5% goes to us (all to accountant while we're low volume)
```